container/deploy_openqa_containers.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+set -e
+
+FORCE=0
+PREFIX="openqa"
+NETWORK="${PREFIX}_net"
+PORT="80"
+ACTION=""
+WEBUI_IMAGE="registry.opensuse.org/devel/openqa/containers15.2/openqa_webui:latest"
+WORKER_IMAGE="registry.opensuse.org/devel/openqa/containers15.2/openqa_worker:latest"
+
+function wait_for_container_ready {
+  container=$1
+
+  for ((count=0; count<30; count++))
+  do
```

Comment:
I'd prefer this style:
```suggestion
  for ((count=0; count<30; count++)); do
```